src/renderer/themes-defaults.ts

```diff
@@ -58,7 +58,7 @@ export const defaultDark: LoadedFiddleTheme = {
   editor: {
     base: 'vs-dark',
     inherit: true,
-    rules: [],
+    rules: [{ token: 'custom-date', foreground: '008800' }],
```

Comment:
Should 008800 be prefixed with a `#` here and below?

Asking because I didn't know it could be omitted! :slightly_smiling_face:  If this is legal, then cool